docs/architecture/maui/preface.md

```diff
@@ -0,0 +1,34 @@
+---
+title: A preface to enterprise architecture for MAUI applications
+description: Guidance for what the book will cover, reference materials, and how to approach the book
+author: michaelstonis
+no-loc: [MAUI]
+ms.date: 06/18/2022
+---
+# Introduction
+
+## Purpose
+
```

Comment:
It's not ideal to have one heading followed immediately by a sub-heading. I suggest getting rid of the Purpose subheading, as this is part of the introduction.
```suggestion
```

Reply:
Great catch. After flipping it back and forth a few times, I think keeping the purpose section reads better.